nix/fix-stylish-haskell/default.nix

```diff
@@ -0,0 +1,13 @@
+{ writeShellScriptBin, fd, stylish-haskell }:
+
+writeShellScriptBin "fix-stylish-haskell" ''
+  # stylish-haskell can fail for some files that it can't parse, and fd will terminate
+  # if any of the exec commands fail. Rather than blacklisting the bad files, we simply
+  # ignore the exit code, which is usually unhelpful.
+  ${fd}/bin/fd \
+    --extension hs \
+    --exclude 'dist-newstyle/*' \
+    --exclude 'dist/*' \
+    --exclude '.stack-work/*' \
+    --exec bash -c "${stylish-haskell}/bin/stylish-haskell -i {} || true"
```

Comment:
Looking at this again, this is not a good way to handle this! I would rather this test fails if `stylish-haskell` fails. If it fails to parse a module, it means that a Language extension is enabled by default in the cabal project that is not listed in the stylish-haskell config, which is a problem to be addressed, not ignored.